jenkins/ci.suse.de/templates/cloud-mkcloud-job-upgrade-disruptive-template.yaml

```diff
@@ -20,7 +20,7 @@
             TESTHEAD=1
             cloudsource=develcloud{previous_version}
             upgrade_cloudsource=develcloud{version}
-            nodenumber=3
+            nodenumber=4
             cephvolumenumber=2
```

Comment:
Well, as you wish. I'd just put it together, no need to make two small PR's in this case